src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -107,7 +108,18 @@ def __init__(self, net_params, batch_size, num_classes):
                 self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
                     net_params[key]))
             else:
-                if 'dense' in key:
+                if 'drawing_dense0_weight' in key:
+                    '''
+                    To make output of MXNET pool3 (NCHW) compatible with TF (NHWC).
```

Comment:
I'm confused by this comment. Do you want mean `Core ML` rather than `MXNET`?

Reply:
I was told that CoreML uses the MXNET format. @fareeha is the original author of this conversion, and I think she has more context than I do.

Reply:
Please don't reference MXNet, which is irrelevant to us now. CoreML does use NCHW

Reply:
changed.